hbase-client/src/main/java/org/apache/hadoop/hbase/client/ReversedClientScanner.java

```diff
@@ -65,6 +58,6 @@ protected boolean setNewStartKey() {
   @Override
   protected ReversedScannerCallable createScannerCallable() {
     return new ReversedScannerCallable(getConnection(), getTable(), scan, this.scanMetrics,
-        this.rpcControllerFactory);
+        this.rpcControllerFactory, scan.getReplicaId());
```

Comment:
The default value for replicaId in Scan is -1. Can we keep the current behavior? i.e, if scan.getReplicaId() < 0, then pass 0, otherwise, use scan.getReplicaId().